packages/mattermost-redux/src/store/initial_state.ts

```diff
@@ -2,6 +2,7 @@
 // See LICENSE.txt for license information.
 
 import {GlobalState} from 'mattermost-redux/types/store';
+import {UserStatuses} from '../../../../utils/constants';
```

Comment:
We can't import code outside of mattermost-redux into the package. Do we not have a version of this constant in mattermost-redux?

Ideally, we wouldn't duplicate those constants, but that's something we need to resolve for a lot more than just the statuses

Reply:
No, we don't have this constant in the redux package. I've updated it to import the `../constants` instead, as was done in another import in the redux package.